docs/examples/index.md

```
@@ -0,0 +1,23 @@
+<script setup lang="ts">
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)